### PR TITLE
CTV-3018

### PR DESCRIPTION
**Pull request by yarodevuci**
- Updated graddle version
- Updated maven url for TAR

### CODE REVIEW

File: ReferenceApp/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,11 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="com.truex.referenceapp">
+    xmlns:tools="http://schemas.android.com/tools">
 
     <application
         android:networkSecurityConfig="@xml/network_security_config"
-        android:allowBackup="true"
+        android:allowBackup="false"
```
Comment (addisonm) [inline ReferenceApp/src/main/AndroidManifest.xml:7]: Good catch.  Though this doesn't get published.  Still worth doing.

File: ReferenceApp/build.gradle [hunk 2]
```diff
@@ -39,11 +39,12 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+    namespace 'com.truex.referenceapp'
 }
 
 repositories {
     maven {
-        url "https://s3.amazonaws.com/android.truex.com/maven"
+        url "https://s3.amazonaws.com/android.truex.com/mobile/prod/maven"
```
Comment (addisonm) [inline ReferenceApp/build.gradle:47]: This is not always going to target prod.  The build scripts are designed to allow QA builds to be released.  See huckleberry mobile and replicate.

Comment (yarodevuci) [reply ReferenceApp/build.gradle:47]: ok i ll add if else statements

**Review by addisonm — CHANGES_REQUESTED** [review]
See comments.

**Review by addisonm — APPROVED** [review]
Please monitor the actual merge to dev and the deployment to S3.

**Pull request merged**